Meshtastic/Helpers/BLEManager.swift

```diff
@@ -869,6 +869,7 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 									traceRoute?.hasPositions = true
 								}
 							}
+							traceRouteHop.name = hopNode?.user?.longName ?? "Unknown"
```

Comment:
Is this overriding an existing node name or is this net new text?

Reply:
The field is currently not used.
I was a bit torn with this one. Without a data model update, we have one name. That could be used for the short name, to correct the name inside the circle. Otherwise, there is this option, to add the long name under the circle.